apps/richie/templates/nginx/secret.yml.j2

```diff
@@ -0,0 +1,12 @@
+{% if activate_http_basic_auth %}
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app: richie
+    service: "nginx"
+  name: "{{ richie_nginx_htpasswd_secret_name }}"
+  namespace: "{{ project_name }}"
+data:
+  "{{ http_basic_auth_user_file | basename }}": "{{ lookup('file', richie_htpasswd) | b64encode }}"
```

Comment:
took me some time to understand wher the "richie_htpasswd" variable comes from, maybe add a comment with link to the line where it's done in `get_vault_for_app`.

Reply:
Yeah, that's a tricky part. Maybe I should make it simpler.